app/src/main/java/fi/bitrite/android/ws/ui/MapFragment.java

```diff
@@ -508,15 +505,10 @@ private void loadOfflineUsers() {
         Disposable mLoadOfflineUserDisposable = Observable.merge(mFavoriteRepository.getFavorites())
                 .filter(Resource::hasData)
                 .map(userResource -> userResource.data)
+                .distinct()
```

Comment:
Because of the map, it's about `userResesource.data`, not `userResource`.  Here, `userResesource.data` is of type `User`, which has an `equal` method implemented. `mOfflineUserIds` can indeed be removed.